### PR TITLE
Allow building container-images.

### PR DESCRIPTION
**Pull request by pilhuhn**
Allow to build container images.
This is off of master and may/will clash with #6 so one or the other will need rebasing

Signed-off-by: Heiko W. Rupp <hrupp@redhat.com>

### CODE REVIEW

File: deploy/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi7-minimal
```
Comment (theute) [inline deploy/Dockerfile]: Any reason why we don't use UBI8 ?

File: deploy/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+LABEL maintainer="hrupp@redhat.com"
+RUN microdnf install java-1.8.0-openjdk-headless && microdnf clean all
```
Comment (theute) [inline deploy/Dockerfile]: And java-11-openjdk-headless ?

File: deploy/Dockerfile (hunk 1 above)
Comment (pilhuhn) [reply deploy/Dockerfile]: I implicitly associated os311 with rhel7 and ubi7. We probably should investigate this.

File: deploy/Dockerfile (hunk 2 above)
Comment (pilhuhn) [reply deploy/Dockerfile]: No specific reason.

File: deploy/Dockerfile (hunk 1 above)
Comment (theute) [reply deploy/Dockerfile]: So since we're running on OS3/RHEL7, the best supported container host would be UBI7.
My question got answered

**Pull request merged**